docs/tutorial/README.md

```diff
@@ -3,8 +3,12 @@
 Often the simplest way to learn about a new technology is to work through some
 tutorials.
 
-You may already have worked your way through the [getting
-started](?id=getting-started) one.
+You may already have worked your way through the [getting started](?id=getting-started) one.
+
+For a much more in depth encounter with qooxdoo, we have some comprehensive tutorials: 
```

Comment:
I wonder - can the tutorial app really be called "comprehensive tutorials"? It's a nice app, no question, but after we have had to remove the twitter tutorial, there isn't really enough left to call it "comprehensive", or "in depth". 

I'd say: "Try out our interactive tutorial app to learn more about how to code with Qooxdoo" or something like that...